kube/secret.go

```diff
@@ -50,9 +60,27 @@ func MakeSecrets(secrets model.CVMap, defaults map[string]string) ([]*apiv1.Secr
 	total := 0 // Accumulated size of the values stored in 'currentSecret'
 
 	for key, value := range secrets {
-		ok, strValue := configValue(value, defaults)
-		if !ok {
-			return nil, nil, fmt.Errorf("Secret '%s' has no value", key)
+		var strValue string
+		if createHelmChart {
+			switch {
+			case value.Generator == nil || value.Generator.Type != model.GeneratorTypePassword:
```

Comment:
No, default was: Inline literal value, encoded in base64.

This is the same as what is being used for the kube config, which requires that *all* dev settings are defined.